tsconfig.base.json

```diff
@@ -1,42 +1,47 @@
 {
-  "compilerOptions": {
-    "baseUrl": ".",
-    "paths": {
-      "~constants": ["src/modules/constants.ts"],
-      "~externalUrls": ["src/modules/externalUrls.ts"],
-      "~admin/*": ["src/modules/admin/components/*"],
-      "~context/*": ["src/context/*"],
-      "~core/*": ["src/modules/core/components/*"],
-      "~dashboard/*": ["src/modules/dashboard/components/*"],
-      "~data/*": ["src/data/*"],
-      "~dialogs/*": ["src/modules/dashboard/components/Dialogs/*"],
-      "~immutable/*": ["src/immutable/*"],
-      "~lib/*": ["src/lib/*"],
-      "~pages/*": ["src/modules/pages/components/*"],
-      "~redux/*": ["src/redux/*"],
-      "~routes/*": ["src/routes/*"],
-      "~styles/*": ["src/styles/shared/*"],
-      "~testutils": ["src/__tests__/utils.ts"],
-      "~modules/*": ["src/modules/*"],
-      "~types/*": ["src/types/*"],
-      "~users/*": ["src/modules/users/components/*"],
-      "~utils/*": ["src/utils/*"]
-    },
-    "allowSyntheticDefaultImports": true,
-    "downlevelIteration": true,
-    "esModuleInterop": true,
-    "jsx": "react",
-    "lib": ["dom", "es2019"],
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "noEmit": true,
-    "noImplicitAny": false,
-    "resolveJsonModule": true,
-    "skipLibCheck": true,
-    "strictNullChecks": true,
-    "strictPropertyInitialization": true,
-    "target": "es6",
-    "typeRoots": ["src/typeDefs", "node_modules/@types"],
-    "types": ["node", "events", "jest"]
-  }
+    "compilerOptions": {
+        "baseUrl": ".",
```

Comment:
It seems that there is an extra space in front of each line vs. the original file. Is it your linting? I think it's good to keep it consistent as in master, without extra space.